analysis_options.yaml

```diff
@@ -11,7 +11,7 @@ linter:
     - avoid_bool_literals_in_conditional_expressions
     - avoid_catches_without_on_clauses
     - avoid_catching_errors
-    - avoid_classes_with_only_static_members
+#    - avoid_classes_with_only_static_members
```

Comment:
to be explicit for future readers pls set to false rather than commenting out

Reply:
Oh yeah forgot to go back and do that, good pickup 👍

Reply:
@maks any idea on how to disable lint rules. Looks like the syntax here in doco is WRONG: https://dart.dev/guides/language/analysis-options#disabling-individual-rules

Throws Dart Analysis errors.

Reply:
`Note: Due to YAML restrictions, you can’t mix list and key-value syntax in the same rules entry`

Reply:
So yeah unfortunately need to either just delete the line with that lint or change all the other rules to use key:clue syntax 😟

Reply:
There seem to be many grumbles on the net about this.

Reply:
OK I converted to the other format and then disabled this lint rule. 👍